.travis.yml

```diff
@@ -57,9 +62,9 @@ script:
 - if [[ ( "$TRAVIS_OS_NAME" == "linux" ) ]]; then gradle test -b test/java/build.gradle; fi
 - ./check-coverage.sh 80
 - npm test --prefix sysl2/sysl/sysl_js
-- GOOS=darwin GOARCH=amd64 go build -o gosysl/gosysl-darwin -ldflags "-X main.Version=${TRAVIS_TAG:1}" github.com/anz-bank/sysl/sysl2/sysl
-- GOOS=linux GOARCH=amd64 go build -o gosysl/gosysl-linux -ldflags "-X main.Version=${TRAVIS_TAG:1}" github.com/anz-bank/sysl/sysl2/sysl
-- go install -ldflags "-X main.Version=${TRAVIS_TAG:1}" github.com/anz-bank/sysl/sysl2/sysl
+- GOOS=${GOOS_DARWIN} GOARCH=${GOARCH} go build -o gosysl/gosysl-darwin -ldflags "-X \"main.Version=${TRAVIS_TAG:1}\" -X \"main.GitCommit=${TRAVIS_COMMIT}\" -X \"main.BuildDate=${BUILD_DATE}\" -X \"main.GoArch=${GOARCH}\" -X \"main.GoOS=${GOOS_DARWIN}\" -X \"main.GoVersion=${GO_VERSION}\" -X \"main.TravisBuildNum=${TRAVIS_BUILD_NUMBER}\" -X \"main.TravisDist=${TRAVIS_DIST}\" -X \"main.TravisOS=${TRAVIS_OS_NAME}\"" -v github.com/anz-bank/sysl/sysl2/sysl
+- GOOS=${GOOS_LINUX}  GOARCH=${GOARCH} go build -o gosysl/gosysl-linux  -ldflags "-X \"main.Version=${TRAVIS_TAG:1}\" -X \"main.GitCommit=${TRAVIS_COMMIT}\" -X \"main.BuildDate=${BUILD_DATE}\" -X \"main.GoArch=${GOARCH}\" -X \"main.GoOS=${GOOS_LINUX}\"  -X \"main.GoVersion=${GO_VERSION}\" -X \"main.TravisBuildNum=${TRAVIS_BUILD_NUMBER}\" -X \"main.TravisDist=${TRAVIS_DIST}\" -X \"main.TravisOS=${TRAVIS_OS_NAME}\"" -v github.com/anz-bank/sysl/sysl2/sysl
+- go install -ldflags "-X \"main.Version=${TRAVIS_TAG:1}\" -X \"main.GitCommit=${TRAVIS_COMMIT}\" -X \"main.BuildDate=${BUILD_DATE}\" -X \"main.GoArch=`uname -m`\" -X \"main.GoOS=`uname -s`\" -X \"main.GoVersion=${GO_VERSION}\" -X \"main.TravisBuildNum=${TRAVIS_BUILD_NUMBER}\" -X \"main.TravisDist=${TRAVIS_DIST}\" -X \"main.TravisOS=${TRAVIS_OS_NAME}\"" github.com/anz-bank/sysl/sysl2/sysl
```

Comment:
Does `uname` output match the expected GOOS and GOARCH values?

Reply:
Nope. I used it in that way as there's no arg is used for GOOS and GOARCH

Reply:
It should match what the values would have been, if specified. You can extract this information from `go version` instead.

Reply:
Thank you! 💡💡